de.json

```diff
@@ -217,7 +217,7 @@
 		"sidebar-collapse": "Zuklappen",
 		"msg-fail-to-save-file": "Fehler beim Speichern der Datei \"{{filepath}}\". {{message}}.",
 		"no-file": "Keine Datei",
-		"switch-vault": "Eine andere Vault öffnen",
+		"switch-vault": "Einen andere Vault öffnen",
```

Comment:
Hmm, do you think 'Vault' should be 'der Vault', 'das Vault' or 'die Vault'?
Personally, 'die Vault' sounds the most natural to me. 
However: I guess the problem is the way we gender words from other languages - some people use the grammatical gender of the traslation (der Tresor -> der Vault, but also: die Kammer -> die Vault - which one should be used?), others use rules derived from the word endings (-ur is mostly feminine, e.g., die Tour), but a rule for the ending '-ault' or something similar does not seem to exist.

Reply:
Oh wow, interesting thought. German is my first language and "der Vault" feels the most natural to me. I asked some Friends and Relatives and they all find "der Vault" the most "correct".

Reply:
I admit I could be biased, because I always associated/translated vault with 'Kammer'.

But I think 'der Vault' is just as valid, and if more people say so - We should then change all the other occurences in another PR, so it is the same everywhere.

Reply:
@FloriF So should we change all occurances to the male (der Vault) version?

Reply:
@fabiobaser Yes, we should try to do this in a new PR^^